iron-overlay-backdrop.html

```diff
@@ -118,10 +118,14 @@
       this.style.zIndex = this._manager.backdropZ();
       // close only if no element with backdrop is left
       if (this._manager.getBackdrops().length === 0) {
+        // Read style before setting opened.
+        var cs = getComputedStyle(this);
+        var noAnimation = (cs.transitionDuration === '0s' || cs.opacity == 0);
```

Comment:
Here we don't wait if `opacity` is already 0, aka nothing's visible already :p
If that's not the case, we patiently wait for any animation to happen ;)
If you want to transition from transparent to opaque, this will do :grin: 
```html
<style is="custom-style">
  iron-backdrop-element {
    --iron-overlay-backdrop-background-color: transparent;
    --iron-overlay-backdrop: {
      transition: background-color 0.3s;
      opacity: 0.6;
    };
    --iron-overlay-backdrop-opened: {
      background-color: black;
    };
  }
</style>
```